runtime/near-test-contracts/build.rs

```diff
@@ -43,10 +43,15 @@ fn build_contract(dir: &str, args: &[&str], output: &str) -> Result<(), Error> {
 
 fn cargo_build_cmd(target_dir: &Path) -> Command {
     let mut res = Command::new("cargo");
-    res.env_remove("RUSTFLAGS");
+
+    res.env_remove("CARGO_BUILD_RUSTFLAGS");
```

Comment:
Why is this needed? Which flags other than `-Dwarnings` are being removed?

Reply:
We don't strictly speaking *need* this I think, but this is sort-of defense in depth, to make the build mode isolated in the face of unforeseen future changes.